test/OpenTelemetry.TraceContext.ValidationTest/Controllers/TestController.cs

```diff
@@ -0,0 +1,69 @@
+﻿// <copyright file="TestController.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+
+namespace OpenTelemetry.TraceContext.ValidationTest.Controllers
```

Comment:
why the existing test app cannot be used here? It already has a proxy controller as far as I remember.

Reply:
They are here: https://github.com/w3c/trace-context/tree/master/test, we should not duplicate them here. I can add link to the code as well

Reply:
good point